ctapipe/calib/camera/r1.py

```diff
@@ -309,6 +313,88 @@ def real_calibrate(self, event):
             self.calibrator.ApplyEvent(samples, fci, r1)
 
 
+class SST1MR1Calibrator(CameraR1Calibrator):
+    '''
+    *  subtract a pixel-wise rough electronic + NSB noise pedestal value,
+        so that the values fluctuate around 0 counts
+        (and record it as MON or SVC data, so it can be un-applied
+        or quality-checked later)
+    *  apply a pixel-wise multiplicative gain correction factor to correct
+        for differences in camera uniformity
+    *  apply a scalar scale and shift the signal so that it efficiently fits
+        in a 16-bit signed integer value with the required dynamic range
+        (see Section 3.3)
+    *  record all applied corrections in SVC or MON streams.
+    '''
+    cell_capacitance_in_farad = 85e-15
+    bias_resistance = 10e3
+    nsb_rate = np.logspace(-3, np.log10(50), 30)
+    gain_drop = (
+        1. / (
+            1. + (
+                nsb_rate *
+                cell_capacitance_in_farad *
+                bias_resistance * 1E9
+            )
+        )
+    )
+
+    # measured in a toy-MC for the above mentioned nsb_rate
+    # the simulation takes some time, so we just put the results here
+    # https://github.com/cta-sst-1m/digicamtoy
+    # TODO: still have to find *the* exact source for these numbers
+    # but it is somewhere in that repo.
+    baseline_shift = np.array([
```

Comment:
I think one could replace this by a formula at some point. We just need to check it with the data first. Could you mark this as a # TODO ?